projects/cn0506/zed/system_bd.tcl

```diff
@@ -0,0 +1,121 @@
+    ad_connect sys_ps7/MDIO_ETHERNET_0 gmii_to_rgmii_0/MDIO_GEM
+    ad_connect sys_ps7/GMII_ETHERNET_1 gmii_to_rgmii_1/GMII
+    ad_connect sys_ps7/MDIO_ETHERNET_1 gmii_to_rgmii_1/MDIO_GEM
+
+    # Remove the unused 200MHz reset generator added in the base design.
+
+    delete_bd_objs [get_bd_nets sys_ps7_FCLK_RESET1_N] \
+                   [get_bd_nets sys_200m_reset] \
+                   [get_bd_nets sys_200m_resetn] \
+                   [get_bd_cells sys_200m_rstgen]
+  }
+  2 {
+    ad_ip_parameter sys_ps7 CONFIG.PCW_QSPI_GRP_SINGLE_SS_ENABLE 1
+    ad_ip_parameter sys_ps7 CONFIG.PCW_ENET0_ENET0_IO EMIO
+    ad_ip_parameter sys_ps7 CONFIG.PCW_ENET0_GRP_MDIO_IO EMIO
+    ad_ip_parameter sys_ps7 CONFIG.PCW_ENET0_PERIPHERAL_FREQMHZ {100 Mbps}
+    ad_ip_parameter sys_ps7 CONFIG.PCW_ENET1_PERIPHERAL_ENABLE 1
+    ad_ip_parameter sys_ps7 CONFIG.PCW_ENET1_GRP_MDIO_ENABLE 1
+    ad_ip_parameter sys_ps7 CONFIG.PCW_ENET1_PERIPHERAL_FREQMHZ {100 Mbps}
+
+    create_bd_port -dir O reset_a
+    create_bd_port -dir O reset_b
+    create_bd_port -dir I ref_clk_50_a
+    create_bd_port -dir I ref_clk_50_b
+    create_bd_intf_port -mode Master -vlnv xilinx.com:interface:rmii_rtl:1.0 RMII_PHY_M_0
+    create_bd_intf_port -mode Master -vlnv xilinx.com:interface:rmii_rtl:1.0 RMII_PHY_M_1
+    make_bd_intf_pins_external  [get_bd_intf_pins sys_ps7/MDIO_ETHERNET_0]
+    make_bd_intf_pins_external  [get_bd_intf_pins sys_ps7/MDIO_ETHERNET_1]
+
+    ad_ip_instance util_mii_to_rmii mii_to_rmii_0
+    ad_ip_parameter mii_to_rmii_0 CONFIG.INTF_CFG 1
+    ad_ip_parameter mii_to_rmii_0 CONFIG.RATE_10_100 0
+
+    ad_connect mii_to_rmii_0/GMII    sys_ps7/GMII_ETHERNET_0
+    ad_connect mii_to_rmii_0/ref_clk ref_clk_50_a
+
+    ad_connect mii_to_rmii_0/RMII RMII_PHY_M_0
+
+    ad_ip_instance util_mii_to_rmii mii_to_rmii_1
+    ad_ip_parameter mii_to_rmii_1 CONFIG.INTF_CFG 1
+    ad_ip_parameter mii_to_rmii_1 CONFIG.RATE_10_100 0
+
+    ad_connect mii_to_rmii_1/GMII    sys_ps7/GMII_ETHERNET_1
+    ad_connect mii_to_rmii_1/ref_clk ref_clk_50_b
+
+    ad_connect mii_to_rmii_1/RMII RMII_PHY_M_1
+
+    ad_ip_instance proc_sys_reset proc_sys_reset_eth0
+    ad_connect proc_sys_reset_eth0/slowest_sync_clk  ref_clk_50_a
+    ad_connect proc_sys_reset_eth0/ext_reset_in  sys_rstgen/peripheral_aresetn
+    ad_connect proc_sys_reset_eth0/peripheral_reset  reset_a
+    ad_connect proc_sys_reset_eth0/peripheral_aresetn  mii_to_rmii_0/reset_n
+
+    ad_ip_instance proc_sys_reset proc_sys_reset_eth1
+    ad_connect proc_sys_reset_eth1/slowest_sync_clk  ref_clk_50_b
+    ad_connect proc_sys_reset_eth1/ext_reset_in  sys_rstgen/peripheral_aresetn
+    ad_connect proc_sys_reset_eth1/peripheral_reset  reset_b
+    ad_connect proc_sys_reset_eth1/peripheral_aresetn  mii_to_rmii_1/reset_n
+  }
+}
+
+source $ad_hdl_dir/projects/scripts/adi_pd.tcl
+
+#system ID
+ad_ip_parameter axi_sysid_0 CONFIG.ROM_ADDR_BITS 9
+ad_ip_parameter rom_sys_0 CONFIG.PATH_TO_FILE "[pwd]/mem_init_sys.txt"
```

Comment:
If the project is parametrized, we should have information regarding what was build, in sysid. This can help in the debug process.
See: https://github.com/analogdevicesinc/hdl/blob/master/projects/dac_fmc_ebz/zcu102/system_bd.tcl#L37

Reply:
Solved in [V2](https://github.com/analogdevicesinc/hdl/pull/919#issuecomment-1180085843)